packages/theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -47,6 +50,17 @@ const HeaderTools = ({
 
   return (
     <PageHeaderTools>
+      <PageHeaderToolsItem>
+        <Text
+          component={TextVariants.a}
+          href="//github.com/patternfly"
+          target="top"
+          aria-label="Link to PatternFly GitHub page"
+          className="ws-github-pageheader pf-u-mr-md"
+        >
+          <GithubIcon />
+        </Text>
```

Comment:
This doesn't take up the full width/height and it's difficult to click on. Maybe go with a button instead like the search icon?
![image](https://user-images.githubusercontent.com/47335686/98720409-7f071280-235e-11eb-8b2a-ff5afcdc7b2b.png)